var/spack/repos/builtin/packages/py-dateparser/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyDateparser(PythonPackage):
+    """dateparser -- python parser for human readable dates"""
+
+    homepage = "https://github.com/scrapinghub/dateparser"
+    url      = "https://pypi.io/packages/source/d/dateparser/dateparser-0.7.2.tar.gz"
+
+    version('0.7.2', sha256='e1eac8ef28de69a554d5fcdb60b172d526d61924b1a40afbbb08df459a36006b')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-convertdate@2.1.3:', type=('build', 'run'))
+    depends_on('py-jdatetime@3.1.0:', type=('build', 'run'))
+    depends_on('py-python-dateutil@2.7.5:', type=('build', 'run'))
+    depends_on('py-pytz@2018.9:', type=('build', 'run'))
+    depends_on('py-regex@2019.01.24:', type=('build', 'run'))
+    depends_on('py-tzlocal@1.5.1:', type=('build', 'run'))
+    depends_on('py-umalqurra@0.2:', type=('build', 'run'))
```

Comment:
Many of these dependencies are optional. I would either remove them, or add a `+calendars` variant and make them optional. I also don't see any of the version restrictions in the `setup.py`. Where do these come from?

Reply:
https://github.com/scrapinghub/dateparser/blob/a08cecce9d57ec88a07687b88fc403bfa5308250/requirements.txt

I just took them from the requirements.txt, iirc this is the one that pip decides on which dependencies to pull in?